### PR TITLE
#106 - add option to show user favorite cities

### PR DESCRIPTION
**Pull request by BarTracz**

### CODE REVIEW

File: app/Http/Controllers/FavoritesController.php [hunk 1]
```diff
@@ -35,4 +35,14 @@ public function check(Request $request, int $cityId): bool
             ->where("city_id", $cityId)
             ->exists();
     }
+
+    public function show(Request $request): array
+    {
+        $userId = $request->user()?->id;
+
+        return Favorites::where("user_id", $userId)
```
Comment (krzysztofrewak) [inline app/Http/Controllers/FavoritesController.php:43]:
```suggestion
        return Favorites::query()->where("user_id", $userId)
```

File: app/Http/Controllers/FavoritesController.php [hunk 2]
```diff
@@ -35,4 +35,14 @@ public function check(Request $request, int $cityId): bool
             ->where("city_id", $cityId)
             ->exists();
     }
+
+    public function show(Request $request): array
```
Comment (krzysztofrewak) [inline app/Http/Controllers/FavoritesController.php:39]: I would prefer to return JsonResponse in controllers. I know that arrays are automatically converted into response objects, but I like to have everything on its right place.

**Review by krzysztofrewak — CHANGES_REQUESTED** [review]

**Comment by vojcc** [comment]
I think we can make something like /my-cities page which would look like thiat:

image

Simple list with map.

**Comment by EwelinaSkrzypacz** [comment]
I'm closing this, because #137 has already solved the issue.

**Pull request closed without merging**